app/javascript/react/assets/styles/Typography/Typography.css

```diff
@@ -0,0 +1,28 @@
+
+@font-face {
+    font-family: "Roboto";
+    font-weight: 400;
+    font-style: normal;
+    font-display: auto;
+    src: url("../../fonts/Roboto-Regular.ttf") format("ttf"),
+      url("../../fonts/Roboto-Regular.ttf") format("ttf");
+  }
+  
+  @font-face {
+    font-family: "Roboto";
+    font-weight: 500;
+    font-style: normal;
+    font-display: auto;
+    src: url("../../fonts/Roboto-Medium.ttf") format("ttf"),
+      url("../../fonts/Roboto-Medium.ttf") format("ttf");
+  }
+
+
+@font-face {
+    font-family: "Roboto";
+    font-weight: 600;
+    font-style: normal;
+    font-display: auto;
+    src: url("../../fonts/RobotoCondensed.ttf") format("ttf"),
```

Comment:
Can we use the same structure for the file name, similar to `-Regular` and `-Medium`?

Reply:
" proszę spójrz jeszcze na te ściągnięte fonty, ich rozszerzenie i te deklaracje font-face’a, bo nie wszystkie są, niech będą tylko woff i woff2 i nazwy tych plików z fontami analogiczne, w SS jest np Roboto-500.woff, Roboto-400.woff itd"